docs/examples/ex_secret_types.py

```diff
@@ -7,17 +7,20 @@ class SimpleModel(BaseModel):
     password_bytes: SecretBytes
 
 sm = SimpleModel(password='IAmSensitive', password_bytes=b'IAmSensitiveBytes')
+
+# Standard access methods will not display the secret
 print(sm)
-#> SimpleModel password=SecretStr('**********') password_bytes=SecretBytes(b'**********')
+# > SimpleModel password=SecretStr('**********') password_bytes=SecretBytes(b'**********')
+print(sm.password.display())
```

Comment:
`.display()` is deprecated after #884 which will be merged before this (try to merge in the order PRs are created unless there's a good reason not to)

Reply:
Changed to `print(str(sm.password))`